static/src/js/schemas/umm/ummCSchema.js

```diff
@@ -2978,27 +3079,135 @@ const ummCSchema = {
+            $ref: '#/definitions/AuthorityType'
+          },
+          Type: {
+            description: 'This element describes to what DOI is associated.',
+            type: 'string',
+            enum: ['Child Dataset', 'Collaborative/Other Agency', 'Field Campaign', 'Parent Dataset', 'Related Dataset']
+          }
+        },
+        required: ['DOI']
+      }, {
+        type: 'object',
+        title: 'Other DOI Types',
+        additionalProperties: false,
+        description: "This element stores the DOI (Digital Object Identifier) that identifies the collection. Note: The values should start with the directory indicator which in ESDIS' case is 10.  If the DOI was registered through ESDIS, the beginning of the string should be 10.5067. The DOI URL is not stored here; it should be stored as a RelatedURL. The DOI organization that is responsible for creating the DOI is described in the Authority element. For ESDIS records the value of https://doi.org/ should be used. NASA metadata providers are strongly encouraged to include DOI and DOI Authority for their collections using CollectionDOI property.",
+        properties: {
+          DOI: {
+            description: "This element stores the DOI (Digital Object Identifier) that identifies the collection.  Note: The values should start with the directory indicator which in ESDIS' case is 10.  If the DOI was registered through ESDIS, the beginning of the string should be 10.5067. The DOI URL is not stored here; it should be stored as a RelatedURL.",
+            type: 'string',
+            minLength: 1,
+            maxLength: 1024
+          },
+          Title: {
+            description: "The title of the DOI landing page. The title describes the DOI object to a user, so they don't have to look it up themselves to understand the association.",
+            $ref: '#/definitions/TitleType'
+          },
+          Authority: {
+            description: 'The DOI organization that is responsible for creating the DOI is described in the Authority element. For ESDIS records the value of https://doi.org/ should be used.',
+            $ref: '#/definitions/AuthorityType'
+          },
+          Type: {
+            description: 'This element describes to what DOI is associated.',
+            type: 'string',
+            enum: ['Other']
+          },
+          DescriptionOfOtherType: {
+            description: 'This element allows the curator to describe what kind of DOI is present when the value of Other is chosen as the type. This element is not allowed if a value other than Other is chosen.',
+            type: 'string',
+            minLength: 1,
+            maxLength: 1024
+          }
+        },
+        required: ['DOI', 'Type', 'DescriptionOfOtherType']
+      }]
+    },
+    OtherIdentifierType: {
```

Comment:
When I select Archive Sets Number (which should have spaces, yes?), and don't fill out identifier, I get the error 'Must have required property 'identifier' twice. Same issue with error 'enum must be equal to one of the allowed values', its unclear what it's really referring to.

Reply:
It also populates the error 'Must have required property 'DescriptionOfOtherType' when i select Type 'ArchiveSetsNumber' which doesn't populate that field.